Corona-Warn-App/src/main/java/de/rki/coronawarnapp/qrcode/ui/info/UqsInfoFragment.kt

```diff
@@ -1,8 +1,37 @@
 package de.rki.coronawarnapp.qrcode.ui.info
 
+import android.os.Bundle
+import android.view.View
+import androidx.core.net.toUri
 import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.google.android.material.transition.MaterialSharedAxis
 import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.FragmentUqsInfoBinding
+import de.rki.coronawarnapp.ui.view.onOffsetChange
+import de.rki.coronawarnapp.util.ui.popBackStack
+import de.rki.coronawarnapp.util.ui.viewBinding
 
 class UqsInfoFragment : Fragment(R.layout.fragment_uqs_info) {
-    // TODO
+
+    private val binding: FragmentUqsInfoBinding by viewBinding()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        enterTransition = MaterialSharedAxis(MaterialSharedAxis.Z, true)
+        returnTransition = MaterialSharedAxis(MaterialSharedAxis.Z, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        binding.apply {
+            toolbar.setNavigationOnClickListener { popBackStack() }
+            privacyInformation.setOnClickListener {
+                findNavController().navigate("coronawarnapp://contactdiary/dataPrivacy".toUri())
```

Comment:
I think you can just use the screen destination Id , they are in the same graph , why do you need to use `uri`
```kotlin
findNavController().navigate(R.id.privacyscreen)`
```

Reply:
You're right, didn't see that.